docs/conf.py

```diff
@@ -56,6 +58,25 @@
 html_theme = 'pydata_sphinx_theme'
 # html_static_path = ['_static']
 
+# -- manage thumbnails --------------------------------------------------------
```

Comment:
Could you please move this to the part of the file under the header for nbsphinx configuration?